### PR TITLE
docs: Add lazy.nvim configration #4694

### PR DESCRIPTION
**Pull request by ElCapitanSponge**
Adding docs for the lazy.nvim configuration as requested in #4694

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,6 +293,16 @@ Plugin 'dense-analysis/ale'
 git clone https://github.com/dense-analysis/ale ~/.vim/bundle/ale
 ```
 
+#### [lazy.nvim](https://github.com/folke/lazy.nvim)
+```lua
+{
+    'desnse-analysis/ale',
+    config = function()
+        -- Configuration goes here.
```
Comment (hsanson) [inline README.md:301]: Better add some more details, for example how to set  specific linters:

```lua
config = function()
  local g = vim.g

  g.ale_ruby_rubocop_auto_correct_all = 1

  g.ale_linters = {
      ruby = {'rubocop', 'ruby'},
  }
end
```

**Review by hsanson — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -293,6 +293,16 @@ Plugin 'dense-analysis/ale'
 git clone https://github.com/dense-analysis/ale ~/.vim/bundle/ale
 ```
 
+#### [lazy.nvim](https://github.com/folke/lazy.nvim)
+```lua
+{
+    'desnse-analysis/ale',
```
[the remark this answers is not in the document]
Comment (ElCapitanSponge) [reply README.md]: I have fixed the typo

File: README.md (hunk 1 above)
Comment (ElCapitanSponge) [reply README.md:301]: I have updated the config function with an example as a starting point to build upon

**Review by hsanson — APPROVED** [review]
Thanks, looks good.

**Pull request merged**